Packs/RecordedFuture/Integrations/RecordedFuture/RecordedFuture.py

```diff
@@ -381,27 +377,28 @@ def build_triage_context(context_data: Dict[str, Any]) \
                       'type': entity['type'], 'score': entity['score'],
                       'Evidence': entity['rule']['evidence']}
                      for entity in context_data['entities']]}
-    indicators = [create_indicator(entity['name'],
-                                   rf_type_to_xsoar_type(entity['type']),
-                                   entity['score'], '')
-                  for entity in context_data['entities']]
-    return context, indicators
+    for entity in context_data['entities']:
+        indicator = create_indicator(entity['name'],
+                                     rf_type_to_xsoar_type(entity['type']),
+                                     entity['score'], '')
+        command_results.append(CommandResults(outputs_prefix='RecordedFuture',
+                                              outputs=context,
+                                              raw_response=context_data,
+                                              readable_output=build_triage_markdown(context_data, context_str),
+                                              outputs_key_field='Verdict',
+                                              indicator=indicator))
```

Comment:
Please separate `context` to a single standalone CommandResults with its own readable_output.
For the indicators, you can create a list of CommandResults with just the `indicator` and a `tableToMarkdown` print of its context.